src/components/private/orders-table-skeleton.tsx

```
@@ -1,40 +1,39 @@
-            <div className="h-4 w-24 animate-pulse rounded bg-gray-200"></div>
-          </TableCell>
-          <TableCell>
-            <div className="h-4 w-24 animate-pulse rounded bg-gray-200"></div>
-          </TableCell>
-        </TableRow>
-      ))}
-    </>
+    Array.from({ length: 10 }).map(() => (
+      <TableRow key={nanoid()}>
+        <TableCell>
+          <div className="size-4 animate-pulse rounded bg-gray-200"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)